### PR TITLE
Initial attempt at using Management API client

### PR DESCRIPTION
**Pull request by emerkle826**

### CODE REVIEW

**Comment by emerkle826** [comment]
@Miles-Garnsey this is just a first shot. You will need to build and compile the Management API client locally before any of this will work for you. And then it still won't work in any CI setup until the client is actually published, But it gives you a very rough idea how the client will work.

File: src/server/pom.xml [hunk 1]
```diff
@@ -271,6 +271,11 @@
             <artifactId>jmxremote_optional-repackaged</artifactId>
             <version>5.0</version>
         </dependency>
+        <dependency>
+          <groupId>com.datastax</groupId>
+          <artifactId>datastax-mgmtapi-client</artifactId>
+          <version>0.1.0-SNAPSHOT</version>
+        </dependency>
```
Comment (emerkle826) [inline src/server/pom.xml:278]: This should be the Maven coordinates for the library once I get it published.

File: src/server/src/main/java/io/cassandrareaper/management/http/HttpCassandraManagementProxy.java [hunk 2]
```diff
@@ -49,6 +59,18 @@ public HttpCassandraManagementProxy(MetricRegistry metricRegistry,
...
+        // with the approrpiate base URL (i.e. "http://localhost:8080")
+        .setBasePath(endpoint.getHostString())
+        // probably want the default 10 second connection timeout
+        //.setConnectTimeout(0)
+        // At the very least, the stopNode operation can take longer than the default 10 second read timeout
+        .setReadTimeout(0)
+        // We might have write operations that take longer than 10 seconds too
+        .setWriteTimeout(0);
+    // setup the helper client with the configured api
+    this.apiClient = new DefaultApi(api);
```
Comment (emerkle826) [inline src/server/src/main/java/io/cassandrareaper/management/http/HttpCassandraManagementProxy.java:73]: The biggest question here is getting the right Base URL for the client to connect. I don't know that `endpoint` here is going to have the protocol prefix (i.e. "http://").

**Review by Miles-Garnsey — APPROVED** [review]
Approving, let's see how this goes.

**Pull request merged**